IndexProactiveControls.md

```diff
@@ -0,0 +1,166 @@
+# Table of contents
+
+<!-- Table of contents generated with markdown-toc https://ecotrust-canada.github.io/markdown-toc/ -->
+
+- [Objective](#objective)
+- [Not adressed CS](#not-adressed-cs)
+- [1. Define Security Requirements](#1-define-security-requirements)
+- [2. Leverage Security Frameworks and Libraries](#2-leverage-security-frameworks-and-libraries)
+- [3. Secure Database Access](#3-secure-database-access)
+- [4. Encode and Escape Data](#4-encode-and-escape-data)
+- [5. Validate All Inputs](#5-validate-all-inputs)
+- [6. Implement Digital Identity](#6-implement-digital-identity)
+- [7. Enforce Access Controls](#7-enforce-access-controls)
+- [8. Protect Data Everywhere](#8-protect-data-everywhere)
+- [9. Implement Security Logging and Monitoring](#9-implement-security-logging-and-monitoring)
+- [10. Handle All Errors and Exceptions](#10-handle-all-errors-and-exceptions)
+
+# Objective
+
+This index have for objective to indicate to help an [OWASP Proactive Controls](https://www.owasp.org/index.php/OWASP_Proactive_Controls) user to clearly identify which cheat sheets are useful for each section during his usage of the OWASP Proactive Controls.
+
+# Not adressed CS
+
+[AJAX Security Cheat Sheet](cheatsheets/AJAX_Security_Cheat_Sheet.md)
```

Comment:
Might this be usable in section 4?

Reply:
It can be usable for section 4 but this CSs have a client side and server side so I am not sure if it should be added here

Reply:
Or similar approach to one that I suggested to `All?` - link to specific section in CS. In this example it will be AJAX_Security_Cheat_Sheet.md#client-side-javascript

Reply:
Yes, I agree to the mentioned method.

Reply:
done

Reply:
"_Not addressed CS_" allow indeed user to raise an issue if they find a coverage missing and ask for a CS to be created.